impeller/image/backends/skia/compressed_image_skia.cc

```diff
@@ -46,12 +47,12 @@ DecompressedImage CompressedImageSkia::Decode() const {
       },
       src);
 
-  auto generator = SkImageGenerator::MakeFromEncoded(sk_data);
-  if (!generator) {
+  auto img = SkImages::DeferredFromEncodedData(sk_data);
```

Comment:
nit: please spell out `image` here and below.

Reply:
Done